### DISCUSSION TITLE
Cloudflare pages / workers

### DISCUSSION THREAD

**Discussion by lewebsimple** [nuxt/nuxt#17843] [Ideas]
With the release of [Workers support in Cloudflare Pages](https://blog.cloudflare.com/cloudflare-pages-goes-full-stack/), it would be interesting to leverage this for ISG deployments, I.e. updating the page cache directly from the worker.
Any thoughts on how to achieve this ?
Would KV / durable objects be better for some reason ?
Are there any good example repos out there ?

**Comment by Rigo-m** [comment]
Adding onto this, any plans for supporting cloudflare pages as a nitro preset? @pi0

**Reply by danielroe** [reply]
Yes, absolutely, but it's dependent on the migration to nitropack: https://github.com/nuxt/framework/issues/3161
